sunpy/net/dataretriever/tests/test_goes_ud.py

```diff
@@ -1,70 +1,54 @@
-    assert ans2 == False
-    ans3 = goes.GOESClient._can_handle_query(Time('2012/8/9', '2012/8/10'), Instrument('eve'))
-    assert ans3 == False
+GClient = goes.GOESClient()
 
 @pytest.mark.online
-def test_query():
-    qr1 = LCClient.query(Time('2012/8/9','2012/8/10'), Instrument('goes'))
-    assert isinstance(qr1,QueryResponse)
-    assert len(qr1) == 2
-    assert qr1.time_range()[0] == '2012/08/09'
-    assert qr1.time_range()[1] == '2012/08/10'
-
+@pytest.mark.parametrize("time, instrument, physobs",
+[(Time(datetime.datetime.now() - datetime.timedelta(days=2), datetime.datetime.now()), Instrument('goes'),
+  Physobs('PARTICLE_FLUX'))])
+def test_query(time, instrument, physobs):
+    qr = GClient.query(time, instrument, physobs)
+    assert len(qr) == 8
 
 @pytest.mark.online
-@pytest.mark.parametrize("time, instrument",
-[(Time('2012/01/17', '2012/01/19'), Instrument('goes')),
- (Time('2012/10/4', '2012/10/6'), Instrument('goes')),
-])
-def test_get(time, instrument):
-    qr1 = LCClient.query(time, instrument)
-    res = LCClient.get(qr1)
-    download_list = res.wait()
-    assert len(download_list) == len(qr1)
+@pytest.mark.parametrize("time, instrument, physobs",
+[(Time(datetime.datetime.now() - datetime.timedelta(days=2), datetime.datetime.now()), Instrument('goes'),
+  Physobs('IRRADIANCE'))])
+def test_query_1(time, instrument, physobs):
+    qr = GClient.query(time, instrument, physobs)
+    assert len(qr) == 16
+
+def test_can_handle_query():
+    assert GClient._can_handle_query(Time('2016/6/15', '2016/6/17'), Instrument('goes'), Physobs('PARTICLE_FLUX'))
+    assert GClient._can_handle_query(Time('2016/6/15', '2016/6/17'), Instrument('goes'), Physobs('INTENSITY'))
+    assert GClient._can_handle_query(Time('2016/6/15', '2016/6/17'), Instrument('goes'), Physobs('IRRADIANCE'))
+    assert not GClient._can_handle_query(Time('2016/6/15', '2016/6/17'), Instrument('goes'))
 
+#Downloads 6 fits files with total
+#size of ~6.5MB
 @pytest.mark.online
-def test_new_logic():
-    qr = LCClient.query(Time('2012/10/4','2012/10/6'), Instrument('goes'))
-    res = LCClient.get(qr)
-    download_list = res.wait()
-    assert len(download_list) == len(qr)
+@pytest.mark.parametrize("time, instrument, physobs",
+[(Time('2016/6/4', '2016/6/4 00:02:00'), Instrument('goes'), Physobs('INTENSITY'))])
+def test_query_2(time, instrument, physobs):
+    qr = GClient.query(time, instrument, physobs)
+    res = GClient.get(qr)
+    d_list = res.wait()
+    assert len(qr) == len(d_list)
 
 @pytest.mark.online
-@pytest.mark.parametrize("time, instrument",
-                         [(a.Time("2012/10/4","2012/10/6"), a.Instrument("goes")),
-                          (a.Time('2013/10/5', '2013/10/7'), a.Instrument("goes"))])
-def test_fido(time, instrument):
-    qr = Fido.search(a.Time('2012/10/4','2012/10/6'), Instrument('goes'))
+def test_fido():
+    qr = Fido.search(Time('2016/6/4', '2016/6/4 00:02:00'), Instrument('goes'), Physobs('INTENSITY'))
```

Comment:
don't remove tests, adapt them, so you should include the two dates above to this tests too. I believe they are irradiance too.